### PR TITLE
version 2 of WCAG-listingpage

### PR DESCRIPTION
**Pull request by nyolarraklay**
## Title
issues [WCAG - Listings page#883](https://github.com/NoroffFEU/agency.noroff.dev/issues/820)

## Describe your changes: 
Resolved the Errors and alerts from WAVE tools
Errors:
1. Empty form label
Alerts:
1. A nearby image has the same alternative text
2. Skipped heading level
3. Redundant link
4. Duplicate ID's

## Type of changes:
1. created an invisible span inside label.
2. changed the alt-text 
3. created an invisible heading
4. created a function that redirect user when they click the button instead of a link
5. changed id to class

## Checklist before requesting a review

* [* ] I created a new branch before starting with this ticket?
* [* ] I commented the issue after finish in the Frontend Team Board?
* [* ] I moved the card to Quality Assurance after finish?

## Did you run into any issues?

- No, issues encountered.

## Screenshots:

- Yes, the screenshots are included in the table below.

| Before | After |
---------|--------
Before:
Skjermbilde 2024-01-16 110249
|        |       |
After:
Skjermbilde 2024-01-16 110219

## Feedback

- No, I don't want feedback.
- Yes, I want feedback.

### CODE REVIEW

**Comment by Sovak3441** [comment]
Closed. The issue will be addressed in another PR.

File: src/js/templates/listings/renderListings.js [hunk 1]
```diff
@@ -98,8 +98,13 @@ const createCardFooter = (deadline) => {
...
     'View',
-    '../../..//pages/listings/listing/index.html'
+    '#'
   );
+  a.addEventListener('click', handleClick);
   element.append(span2, a);
   return element;
 };
+function handleClick() {
+  window.location.href = "../../..//pages/listings/listing/index.html"
```
Comment (Spookyrumble) [inline src/js/templates/listings/renderListings.js:108]: I don't think this href will pass. there is 2 / before "pages"

**Review by Spookyrumble — CHANGES_REQUESTED** [review]
`<h2 class="visually-hidden">Subheading 1</h2>
 <h3 class="visually-hidden">Subheading 1</h3>
 <h4 class="visually-hidden">Subheading 1</h4>`

I don't understand the reason for this? 

The hrefs in renderListings.js needs to be corrected.

The rest LGTM :)

File: src/js/templates/listings/renderListings.js (hunk 1 above)
Comment (nyolarraklay) [reply src/js/templates/listings/renderListings.js:108]: i fixed it. thanks for noticing

**Comment by Sovak3441** [comment]
LGTM :) We'll do a merge when @Spookyrumble is happy with the changes he requested :)

**Review by Spookyrumble — APPROVED** [review]
LGTM

**Pull request merged**